apacheconfig/__init__.py

```diff
@@ -6,18 +6,24 @@
 from apacheconfig.lexer import make_lexer
 from apacheconfig.parser import make_parser
 from apacheconfig.loader import ApacheConfigLoader
+from apacheconfig.wloader import ApacheConfigWritableLoader
 from apacheconfig.error import ApacheConfigError
-from apacheconfig.wloader import make_writable_loader  # noqa: F401
 from apacheconfig import flavors  # noqa: F401
 
 
 @contextmanager
-def make_loader(**options):
+def make_loader(writable=False, **options):
```

Comment:
nit: do we need `writable` as a kwarg rather then buried in `options`?

Reply:
I added it as an extra kwarg to disambiguate it from other `options`-- it's really a parameter that is passed into `make_loader`, and not, for instance, a CLI option.

Reply:
My uneducated impression is that may be the best outcome would be to eventually have just one `ApacheConfigLoader` that would do write if asked to, otherwise behaving roughly the same as legacy loader. Then we'd not need this flag at all.

Does it make sense?